splash/request_middleware.py

```diff
@@ -72,6 +72,59 @@ def process(self, request, render_options, operation, data):
         return request
 
 
+class ContentTypeMiddleware(object):
```

Comment:
it is odd to have a response middleware in a file named request_middleware. Let's move it to response_middleware.py